extensions-ocp-rhel-9.4.yaml

```diff
@@ -15,7 +15,7 @@ extensions:
   # https://github.com/coreos/fedora-coreos-tracker/issues/1504
   ipsec:
     packages:
-      - libreswan
+      - libreswan-4.6
```

Comment:
Can you add a comment just above this with a link to the Jira ticket so it's easier to tell when to revert this change?

Reply:
ticket for reverting the change made in this PR: https://issues.redhat.com/browse/SDN-5544